src/components/Comments/Comment.js

```diff
@@ -263,7 +322,7 @@ const Comment = ({
         <TouchableOpacity activeOpacity={1} onPress={onDownVote} testID="btnDownvote">
           <ButtonHightlight
             onLongPress={handleOnLongPress}
-            style={[styles.arrowup, styles.btn]}
+            style={[styles.arrowup, styles.btn, {marginRight: 8, paddingRight: 0}]}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for displaying the total vote count has been removed. Ensure this change doesn't affect the UI when there are no votes.

```diff
- {totalVote !== 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
+ <Text style={styles.vote(voteStyle())}>{totalVote}</Text>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->